docs/usage/basics/basic-configuration.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Basic Shoot configurations
+weight: 6
+---
+
+The [gardener dashboard](https://github.com/gardener/dashboard/blob/master/README.md) provides a simple way for creating and configuring a basic shoot cluster. You can see how changing the shoot configuration through the overview section reflects on the shoot yaml resource. When talking about clusters in the documentation the yaml structure will be mostly refered and therefore a basic understanding of it's structure will be highly usefull.
```

Comment:
I recall raising the same question already many times to @n-boshnakov . "shoot" vs "Shoot" in the docs? "gardener dashboard" vs "Gardener dashboard"? Right now Shoot is with upper-case in the titles but lower-case in this docs.